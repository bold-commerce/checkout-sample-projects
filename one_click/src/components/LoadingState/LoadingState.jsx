import React from 'react';
import './LoadingState.css';
import LoadingSpinner from '@boldcommerce/stacks-ui/lib/components/loadingspinner/LoadingSpinner';

const LoadingState = () => (
  <div className="FieldSet--LoadingState" role="alert">
    <LoadingSpinner />
  </div>
);

export default LoadingState;
