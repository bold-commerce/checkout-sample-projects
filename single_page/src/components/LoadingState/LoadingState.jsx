import React from 'react';
import { LoadingSpinner } from '@boldcommerce/stacks-ui';
import './LoadingState.css';

const LoadingState = () => (
  <div className="FieldSet--LoadingState">
    <LoadingSpinner />
  </div>
);

export default LoadingState;
