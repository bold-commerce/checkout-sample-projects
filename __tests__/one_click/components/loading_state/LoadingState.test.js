import React from 'react';
import LoadingState from '../../../../one_click/src/components/LoadingState/LoadingState';
import { render } from '@testing-library/react';

describe('LoadingState', () => {
    test('renders LoadingState component', () => {
        const { asFragment } = render( <LoadingState /> );
        expect(asFragment()).toMatchSnapshot();
    });
});
