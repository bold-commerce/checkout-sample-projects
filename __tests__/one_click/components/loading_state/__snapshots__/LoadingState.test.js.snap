// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingState renders LoadingState component 1`] = `
<DocumentFragment>
  <div
    class="FieldSet--LoadingState"
  >
    <span
      class="stx-loading-spinner"
    />
  </div>
</DocumentFragment>
`;
